CHANGELOG.md

```diff
@@ -6,7 +6,11 @@
 
 #### Enhancements
 
-* None.
+* Adds `--force-exclude` option to `lint` and `autocorrect` commands, which will
+  force SwiftLint to exclude files specified in the config `excluded` even if
+  they are explicitly specified with `--path`.
```

Comment:
Can you add two trailing whitespaces after `.`?

Reply:
Sure can. Would you like a separate PR that checks for these missing spaces using Danger? Could save you some time chasing after contributors 😄